java/src/com/google/idea/blaze/java/fastbuild/FastBuildBlazeData.java

```diff
@@ -224,13 +224,16 @@ abstract static class JavaToolchainInfo {
 
     public abstract String targetVersion();
 
+    public abstract Optional<JavaRuntime> javaRuntime();
+
     static JavaToolchainInfo create(
         ImmutableList<ArtifactLocation> javacJars,
         ImmutableList<ArtifactLocation> bootJars,
         String sourceVersion,
-        String targetVersion) {
+        String targetVersion,
+        Optional<JavaRuntime> javaRuntime) {
```

Comment:
Idiomatic use of Optional type in Java is only for return types

Reply:
fixed please see new commit